client/src/partials/development/reports/generate/patient_invoices.generate.js

```diff
@@ -0,0 +1,101 @@
+// Controller name must match key convention over BHIMA style guide
+angular.module('bhima.controllers')
+.controller('patient_invoicesController', patientInvoices);
+
+// Definition is passed in through report core
+patientInvoices.$inject = ['$anchorScroll', '$location', '$modalInstance', '$http', 'util', 'store', 'ModuleState', 'validate', 'definition', 'updateMethod', 'referenceOptions'];
```

Comment:
These injects could be cleaned up to reflect what you are actually using in the controller (makes it easier to review).  Ditto for all the other controllers that are templated on-the-fly.